springwolf-examples/springwolf-kafka-example/src/test/java/io/github/stavshamir/springwolf/example/OpenApiGeneratorTest.java

```diff
@@ -0,0 +1,29 @@
+package io.github.stavshamir.springwolf.example;
+
+import org.json.JSONException;
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.testcontainers.shaded.org.apache.commons.io.IOUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+
+public class OpenApiGeneratorTest {
```

Comment:
Maybe put a comment that this test is linked to the build.gradle configuration and plugin?
Just to make the link more obvious

Reply:
I tried to put a comment to explain the connection between the gradle configuration and this test. Does it help?